tools/progress.py

```diff
@@ -41,7 +41,13 @@ def getProgressFromMapFile(mapFile: mapfile_parser.MapFile, asmPath: Path, nonma
                 extensionlessFilePath = extensionlessFilePath.with_suffix("")
 
             fullAsmFile = asmPath / extensionlessFilePath.with_suffix(".s")
-            wholeFileIsUndecomped = fullAsmFile.exists()
+
+            handwrittenAsmFiles = [Path("boot/libc64/fp.o"), Path("makerom/entry.o")]
+            
+            if originalFilePath in handwrittenAsmFiles:
```

Comment:
Part of me wonders if we should just skip counting the handwritten instead of just counting them as done? Would cause a weird dip in a possible graph though.

Reply:
I tried this but I think the issue is that makerom will no longer have any bytes, which causes mapfile_parser.progress_stats.printStats() to give a divide by zero error